CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -380,43 +386,43 @@ def main(argv):
                     baseColumnIdx = None
-                    if args.baseColumn:
+                    if baseColumn:
                         for colIdx in defaultColumnDict.keys():
-                            if defaultColumnDict[colIdx] == args.baseColumn:
+                            if defaultColumnDict[colIdx] == baseColumn:
                                 baseColumnIdx = colIdx 
 
                     ## Check row for mismatch and print results
-                    rowCheckResults = checkRowForMismatch(row, defaultColumnDict, baseColumnIdx, args.ignoreOrder, wsOut, mismatchFlagIdx, args.outputMismatchTypesFlag, args.formatCheckFlag, args.formatCheckCharactersAdd, args.formatCheckCharacters)
+                    rowCheckResults = checkRowForMismatch(row, defaultColumnDict, baseColumnIdx, ignoreOrder, wsOut, mismatchFlagIdx, outputMismatchTypesFlag, formatCheckFlag, formatCheckCharactersAdd, formatCheckCharacters, verbose)
                     if len(rowCheckResults[1]) > 0:
                         if ws.title not in wsMismatchDict.keys():
                             wsMismatchDict[ws.title] = 1
                         else:
                             wsMismatchDict[ws.title] += 1
-                        if args.verbose:
+                        if verbose:
                             baseColumnName = defaultColumnDict[list(rowCheckResults[0].keys())[0]]
-                            if args.outputMismatchTypesFlag:
+                            if outputMismatchTypesFlag:
                                 mismatchColumnNames = ",".join("%s (%s)" % (defaultColumnDict[i], ",".join(rowCheckResults[1][i][1])) for i in rowCheckResults[1].keys())
                             else:
                                 mismatchColumnNames = ",".join(defaultColumnDict[i] for i in rowCheckResults[1].keys())
                             print("WARNING %s row %s: the output values in %s do not match %s" % (ws.title, rowIdx+2, mismatchColumnNames, baseColumnName))
-            elif args.verbose:
+            elif verbose:
                 print("WARNING %s: No columns found for comparison" % (ws.title,))
 
             ## If ws is a configuration sheet, run the configuration check
-            if ws.title == args.configurationSheet:
-                wbMissingSheets = checkConfigurationSheet(wb, ws, args.configurationSheetColumnName, wsOut, args.verbose)
+            if ws.title == configurationSheet:
+                wbMissingSheets = checkConfigurationSheet(wb, ws, configurationSheetColumnName, wsOut, verbose)
         except Exception as e:
             print("FATAL ERROR in worksheet %s : %s" % (ws.title, str(e)))
-            if args.debugMode:
+            if debugMode:
                 tb.print_exc(e)
             exit(-1)
 
     ## Save workbook and print summary
     if len(wsMismatchDict) > 0 or (wbMissingSheets is not None and len(wbMissingSheets) > 0):
-        if args.createOutputFileFlag:
+        if args and createOutputFileFlag:
```

Comment:
I think `args and ` can be removed here

Reply:
`createOutputFileFlag` by default is true and i want the things done under this block to be skipped if we are not running this via command line hence added the check for args.